src/install.ts

```diff
@@ -17,9 +17,10 @@ export async function runCommand(command: string): Promise<string> {
 }
 
 async function installOnLinux(version: string): Promise<string> {
-  const system = runCommand('uname -s')
+  const system = await runCommand('uname -s')
   const hardware = runCommand('uname -m')
-  const url = `https://github.com/docker/compose/releases/download/${version}/docker-compose-${await system}-${await hardware}`
+  const casedSystem = parseInt(version.split('.')[0], 10) >= '2' ? `${system[0].toUpperCase()}${system.slice(1)}` : system
```

Comment:
let's keep type-safety :)

```suggestion
  const casedSystem = parseInt(version.split('.')[0], 10) >= 2 ? `${system[0].toUpperCase()}${system.slice(1)}` : system
```

Reply:
Sure, my fault